.github/workflows/node.js.yml

```
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.15.0
+          node-version: 18.18.0
```

I think you can avoid hardcoding the version by using this other way...

```suggestion
          node-version-file: '.nvmrc'
```

I will take the version from the `nvmrc` file.